### ISSUE TITLE
Error when destroying emitters and no radio menu exists

### ISSUE THREAD

**Issue by davidp57** [uriba107/DCS-Hound#7]
That's the case when the controller has not been enabled.
In `HoundElint:removeRadarRadioItem()`, there is an error on `self.radioMenu.data[assigned].data`.
I added this check to make it work : 
```lua
        if self.radioMenu.root == nil then
            return
        end
```

**Comment by uriba107** [comment]
Hi David,
when exactly does this happen?
when you enable Hound, it adds something to the menu and then you disable it? I understood the problem correctly?

**Comment by davidp57** [comment]
I've made a wrapper for your code, and it makes using a controler optional.
Therefore, if no controler is enabled (i.e. the `HoundElint:enableController()` function is never called), each time Hound notices that an emitter has been destroyed, it will throw an error because `self.radioMenu.data[assigned].data` references `self.radioMenu.data`, which is nil.

**Comment by uriba107** [comment]
Makes sense, I'll get it fixed. Thanks for clearing it up.

**Comment by uriba107** [comment]
Fixed in 0.1.1
